pkg/kubelet/server/server.go

```diff
@@ -210,10 +210,17 @@ func ListenAndServeKubeletReadOnlyServer(
 }
 
 // ListenAndServePodResources initializes a gRPC server to serve the PodResources service
-func ListenAndServePodResources(socket string, podsProvider podresources.PodsProvider, devicesProvider podresources.DevicesProvider, cpusProvider podresources.CPUsProvider, memoryProvider podresources.MemoryProvider) {
+func ListenAndServePodResources(
```

Comment:
I have another PR baking which wants to repack the arguments here in a struct because this is getting out of hand. I'll publish ASAP and perhaps we can spin off this tiny change, merge it quickly and both of us benefit

Reply:
https://github.com/kubernetes/kubernetes/pull/115852/commits/9f4a3bf2721b09d3f616fb5cfa744e95442c46ab

Reply:
sure sound good

Reply:
I cherrypicked https://github.com/kubernetes/kubernetes/pull/115852/commits/71be7c60bd880606af3a97a551c18d2f7212af5f